runtime/runtime/src/genesis.rs

```diff
@@ -21,9 +21,13 @@ use near_store::{
 
 use crate::config::RuntimeConfig;
 use crate::Runtime;
-
+/// Computes the expected storage per account for a given set of StateRecord(s).
```

Comment:
```suggestion
/// Computes the expected storage per account for a given stream of StateRecord(s).
```

the fact that we do streaming is salient here -- otherwised this would be a function.

Reply:
done.